test/type.query.test.js

```diff
@@ -277,4 +277,22 @@ describe('type: query', () => {
             return;
         });
     });
+    describe('Execute ================', () => {
+        it('Should start executing queries', async () => {
+            const execute = await handler.execute('testInstance/testBU', ['query']);
+            assert.equal(process.exitCode, false, 'execute should not have thrown an error');
+            assert.equal(execute, true, 'queries were supposed to be executed');
+            return;
+        });
```

Comment:
this test will need to be removed or changed to execute 2+ specific queries - and test for a different return value